jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -0,0 +1,806 @@
+----
+global:
+  domainName: "<full-domain-name-of-your-install>"
+  license: '<license-for-circleci-server>'
+
+apiToken: "<circleci-api-token>"
+sessionCookieKey: "<session-cookie-key>"
+keyset:
+  signing: '<generated-signing-key>'
+  encryption: '<generated-encryption-key>'
+
+github:
+  clientId: "<generated-github-client-id>"
+  clientSecret: "<generated-github-client-secret>"
+object_storage:
+  bucketName: "<gcs-bucket-name>"
+  gcs:
+    enabled: true
+    workloadIdentity: "<service-account-email-with-gcs-access>"
+
+mongodb:
+  auth:
+    rootPassword: "<mongodb-root-password>"
+    password: "<mongodb-password>"
+vm_service:
+  providers:
+    gcp:
+      enabled: true
+      project_id: <gcp-project-id>
+      network_tags:
+      - <network-tag>
+      zone: <gcp-zone>
+      network: "<gcp-network>"
+      subnetwork: "" # leave blank for auto-subnetting
+      workloadIdentity: "<service-account-email-with-compute-access>"
+
+pusher:
+  secret: "<pusher-secret>"
+postgresql:
+  auth:
+    postgresPassword: "<postgres-password>"
+rabbitmq:
+  auth:
+    password: "<rabbitmq-password>"
+    erlangCookie: "<rabbitmq-erlang-cookie>"
+nomad:
+  server:
+    gossip:
+      encryption:
+        key: "<nomad-gossip-encryption-key>"
+    rpc:
+      mTLS:
+        enabled: true
+        CACertificate: "<nomad-mtls-base64-ca>"
+        certificate: "<nomad-mtls-base64-cert>"
+        privateKey: "<nomad-mtls-base64-key>"
+----
+
+[#deploy]
+== 4. Deploy
+
+Once you have completed the fields detailed above, you can deploy CircleCI's core services:
+
+[source,shell]
+----
+helm install server -f values.yaml <path-to-helm-chart>
+----
+
+[#create-dns-entry]
+== 5. Create DNS entry
+Create a DNS entry for your nginx load balancer, for example, `circleci.your.domain.com` and `app.circleci.your.domain.com`. The DNS entry should align with the DNS names used when creating your TLS certificate and GitHub OAuth app during the prerequisites steps. All traffic will be routed through this DNS record.
+
+You need the IP address, or, if using AWS, the DNS name of the nginx load balancer. You can find this information with the following command:
+
+[source,shell]
+----
+kubectl get service circleci-proxy
+----
+
+For more information on adding a new DNS record, see the following documentation:
+
+* link:https://cloud.google.com/dns/docs/records#adding_a_record[Managing Records] (GCP)
+
+* link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-creating.html[Creating records by using the Amazon Route 53 Console] (AWS)
+
+[#validation]
+== 6. Validation
+
+You should now be able to navigate to your CircleCI server installation and log in to the application successfully.
+
+Now we will move on to build services. It may take a while for all your services to be up. You can periodically check by running the following command (you are looking for the “frontend” pod to show a status of _running_ and **ready** should show 1/1):
```

Comment:
Is there a reason one of these is italics and one is bold in "status of _running_ and **ready** "?

Reply:
I will find out!